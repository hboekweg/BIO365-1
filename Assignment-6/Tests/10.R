source("10.R")