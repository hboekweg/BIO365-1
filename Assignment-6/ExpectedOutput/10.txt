# A tibble: 3 x 5
  hurricane  year   mph Latitude Longitude
  <chr>     <dbl> <dbl>    <dbl>     <dbl>
1 Gilbert    1988   186     17.7      63.3
2 Maria      2017   175     20.0      67.2
3 Noel       2007    80     18.5      66.1
